src/javascripts/components/tabs.js

```diff
@@ -43,32 +43,42 @@ AppTabs.prototype.activateAndToggle = function (event) {
   event.preventDefault()
   var $currentToggler = event.target
   var $currentTogglerSiblings = this.$module.querySelectorAll('[href="' + $currentToggler.hash + '"]')
-  var $tabContainer = this.$module.querySelector($currentToggler.hash)
+  var $tabContainer
+
+  try {
+    $tabContainer = this.$module.querySelector($currentToggler.hash)
+  } catch (exception) {
+    if ((typeof console !== 'undefined' || typeof console.log !== 'undefined')) {
+      console.error('Invalid example ID given: + ' + exception)
+    }
+  }
   var isTabAlreadyOpen = $currentToggler.getAttribute('aria-expanded') === 'true'
 
-  if (isTabAlreadyOpen) {
-    $tabContainer.classList.add(tabContainerHiddenClass)
-    $tabContainer.setAttribute('aria-hidden', 'true')
-    nodeListForEach($currentTogglerSiblings, function ($tabToggler) {
-      $tabToggler.setAttribute('aria-expanded', 'false')
-      // desktop and mobile
-      $tabToggler.parentNode.classList.remove(tabsItemCurrentClass, headingItemCurrentClass)
-    })
-  } else {
-    // Reset tabs
-    this.resetTabs()
-    // make current active
-    $tabContainer.classList.remove(tabContainerHiddenClass)
-    $tabContainer.setAttribute('aria-hidden', 'false')
+  if ($tabContainer) {
```

Comment:
To avoid unnecessary nesting you could consider returning early here, although I haven't seen the rest of the code so might not be possible :)

Reply:
Hmmm my reasoning was that it's often better to stick to a single `return` point to make the code more readable. But I think in this case the readability increased with your equally valid suggestion so have changed it. Thanks 🙂